packages/services/f-analytics/src/services/_tests/analytics.service.test.js

```diff
@@ -348,20 +348,20 @@ describe('Analytic Service ::', () => {
             }
         );
 
-        it('should set conversationId and requestId values when provided', () => {
+        it('should set requestId value when provided', () => {
             // Arrange
             const expected = {
                 ...defaultState.pageData,
                 group: 'test-feature-name',
```

Comment:
Just wondering if it's necessary to pass most of these values in. Could they not just be the defaults, and we only override the values that are pertinent to this particular test?

Reply:
This is the default state `...defaultState.pageData` and is used elsewhere in various tests, so he could create a new expected state which is pretty much what he is doing here.